dts/bindings/led/ti,tlc59731.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2020 Seagate Technology LLC
```

Comment:
Is this the correct Copyright?

Reply:
> Is this the correct Copyright?

Thanks, not it is not.  I forgot to add my own